index.html

```diff
@@ -1,23 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<link rel="reset" type="text/css" href="css/reset.css">
+		<link href='<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>'>
+		<link href="css/StyleSheet.css" rel="StyleSheet"/>
     	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
-    	<meta name="keywords" content="one,two,three">
-
-		<title>Blank Template</title>
-
-		<!-- CSS links -->
-
+    	<meta name="Relaxr Website" content="Relaxr landing page">
+    	<meta name="Relaxr, landin page" content="Technology, Lifestyle, Automation">
+		<title>Relaxr</title>
 	</head>
 	<body>
-		<div>
+		<div id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<h1 id="TopLogo">Relaxr</h1>
+				<h2>Get piece of mind with a single tap</h2>
+				<input type="button" value="Get it Now">
```

Comment:
Maybe a `<button>` element would be a better choice for this. `input` tags are generally used inside of `form` elements.